index.html

```diff
@@ -0,0 +1,156 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>Product Landing Page</title>
+</head>
+
+<body>
+  <header>
+    <nav>
+      Explore
+      Start a project
+      <img src="logo.png">
+      <input type="text" placeholder="Search">
+      Log in
+    </nav>
+    <h1>The Minimalist’s Wall Calendar | Visualize Your Year</h1>
+    <p>Minimalist, forgiving, and beautiful. A wall calendar designed to make 2021 feel like a breath of fresh air.</p>
+  </header>
+  <main>
+    <figure>
+      <img src="hero.png">
+      <figcaption>Project we Love | Product Design | Cambridge, MA</figcaption>
+    </figure>
+    <aside>
+      <hr>
+      <p>
+        $13,757<br>
+        pledged of $1,200 goal
+      </p>
+      <p>
+        215 <br>
+        backers
+      </p>
+      <p>
+        19 <br>
+        days to go
+      </p>
+      <input type="button" value="Back this Project"><br>
+      <input type="button" value="Remind me">
+      <p>
+        <sub>All or nothing. This project will only be funded if it reaches its goal by Thu, November 26 2020 1:35 PM
+          CST.</sub>
+      </p>
+    </aside>
+    <nav>
+      Campaign | FAQ | Updates<sup>2</sup> | Comments<sup>19</sup> | Community
+    </nav>
```

Comment:
Similar comment to the main `<nav>` tag above. It looks like you intended to treat each of these items (e.g. Campaign, FAQ, Updates, etc.) as individual elements in the nav. But right now they're a _single_ element.

You'll want to wrap them in their own HTML tags to make them their own elements on the page.